src/vs/workbench/services/extensions/common/abstractExtensionService.ts

```diff
@@ -464,18 +469,24 @@ export abstract class AbstractExtensionService extends Disposable implements IEx
 			const affinity = affinities.get(ExtensionIdentifier.toKey(extension.identifier)) || 0;
 			result.set(ExtensionIdentifier.toKey(extension.identifier), new LocalProcessRunningLocation(affinity));
 		}
+		const { affinities: localWebWorkerAffinities, maxAffinity: maxLocalWebWorkerAffinity } = this._computeAffinity(localWebWorkerExtensions, ExtensionHostKind.LocalProcess, isInitialAllocation);
```

Comment:
Should we pass in here `ExtensionHostKind.LocalWebWorker` instead of `ExtensionHostKind.LocalProcess`?

Reply:
Absolutely! Thank you 🙇